.ado/continuous.yml

```diff
@@ -8,27 +8,6 @@ variables:
   - group: platform-override-zero-permission-token
 
 stages:
-  - stage: Setup
-    jobs:
-      - job: Setup
-        variables:
```

Comment:
Why is variables not used in the new setup.yml?

Reply:
Same as PR setup. We no longer use and of the variables (in environment or otherwise) that are set.